main.py

```diff
@@ -160,6 +167,21 @@ def scrape(self, site, depth=0):
             content_type = self.get_content_type(headers)
             print("link", href, content_type)
             # if this is a document type, store it
+
+            if new:
+                if self.total_new_gdoc_count >= MAX_NEW_GOOGLE_DOCS:
+                        break
```

Comment:
This breaks out of the for loop as soon as we reach the max for either google drive or regular files (ie if we hit 30 google drive files, it will not continue scraping regular files even if they are below 100).  Is this the desired behavior?